<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>salle_page</class>
 <widget class="QDialog" name="salle_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/new/prefix3/beige.jpg);</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>851</width>
     <height>431</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix3/orange.png);</string>
   </property>
   <property name="title">
    <string>Gestion salle</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QTabWidget" name="tabWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>831</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix2/marron.jpg);</string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab_13">
     <attribute name="title">
      <string>Ajout</string>
     </attribute>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Numero :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Nombre de place :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Nombre de place disponible :</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_nb_place_salle_ajout">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>70</y>
        <width>471</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: white</string>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_nb_place_dispo_salle_ajout">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>120</y>
        <width>471</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: white</string>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_numero_salle_ajout">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>501</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: white</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_ref_cinema">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>180</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: white</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_load_reference">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>180</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix3/orange.png);</string>
      </property>
      <property name="text">
       <string>Charger</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Référence du cinema :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ajouter_salle">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>250</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix3/orange.png);
/*border-radius :60px;*/
border-radius:25px;










</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_14">
     <attribute name="title">
      <string>Affichage liste</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_affichage_salle">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>381</width>
        <height>271</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>18</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;
border-image: url(:/new/prefix3/orange.png);
border-radius:25px;
</string>
      </property>
      <property name="text">
       <string>Affichage</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_affichage_list_salle">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>40</y>
        <width>401</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: white</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_15">
     <attribute name="title">
      <string>Modification</string>
     </attribute>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Nombre de place :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Nombre de place disponible :</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_affichage_list_salle_2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>70</y>
        <width>361</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: white</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_modifier_salle">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix3/orange.png);
border-radius:25px;
</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_charger_salle">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>240</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix3/orange.png);
border-radius:25px;
</string>
      </property>
      <property name="text">
       <string>Charger</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nombre_salle_modification">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: white</string>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nombre_salle_disponible_modifcation">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>130</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: white</string>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>20</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">text-decoration: underline;
color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string>Numero :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_numero_salle_modifcation">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>20</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: white</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_16">
     <attribute name="title">
      <string>Suppression</string>
     </attribute>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Numero :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_numero_salle_supprimer">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: white</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supprimer_salle">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>110</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix3/orange.png);
border-radius:25px;
</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_17">
     <attribute name="title">
      <string>Rechercher</string>
     </attribute>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Numero :</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_salle_rechercher">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>20</y>
        <width>411</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: white</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Référence du cinema :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Nombre de place :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_numero_rechercher">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: white</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="placeholderText">
       <string>Chercher</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ref_cinema_salle_rechercher">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: white</string>
      </property>
      <property name="maxLength">
       <number>8</number>
      </property>
      <property name="placeholderText">
       <string>Chercher</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nombre_place_rechercher">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: white</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="placeholderText">
       <string>Chercher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rechercher_salle">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix3/orange.png);
border-radius:25px;
</string>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
